locales/en/index.yml

```
@@ -213,6 +213,8 @@ authentication:
     card5-content: In this experimental version you can find in the upper-right corner
       the icons to report bugs or start a chat with the development team.
       We're waiting for your feedbacks!
+    ok: Ok
+    cardOk: Ok, your card is valid! Wait a few more seconds ...
```

```suggestion
    cardOk: Ok, your card is valid! Wait a few more seconds..
```
https://projects.invisionapp.com/share/FZUE5VC8JNQ#/screens